etl/steps/data/garden/who/2023-08-14/avian_influenza_ah5n1.meta.yml

```diff
@@ -0,0 +1,12 @@
+tables:
+  avian_influenza_ah5n1:
+    variables:
+      avian_cases:
+        title: Human cases with highly pathogenic avian influenza A/H5N1 (monthly)
+        unit: cases
+        description: >-
+          Monthly number of human cases with highly pathogenic avian influenza A/H5N1.
```

Comment:
Totally. Might consider having it later, actually.

The thing is, I am using this `zeroDay` thing, which applies to all indicators of the dataset (`date` is the number of days instead of year). Hence, the year data looks odd (data for 1st January 2022, etc.). 

It is also not ideal for months tho.